aspnetcore/grpc/loadbalancing.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Scalable apps with gRPC client-side load balancing
+author: jamesnk
+description: Learn how to make scalable, high-performance gRPC apps with client-side load balancing in .NET.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: jamesnk
+ms.date: 05/18/2021
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: grpc/loadbalancing
+---
+# Scalable apps with gRPC client-side load balancing
+
+By [James Newton-King](https://twitter.com/jamesnk)
+
+gRPC client-side load balancing is a feature that allows gRPC clients to distribute load optimally across available servers. This article discusses how to configure a client-side load balancing to create scalable, high-performance gRPC apps in .NET.
```

Comment:
```suggestion
gRPC client-side load balancing is a feature that allows gRPC clients to distribute loads optimally across available servers. This article discusses how to configure a client-side load balancing to create scalable, high-performance gRPC apps in .NET.
```

Reply:
Hmmm, I'm not sure about this. "load" in this context doesn't seem pluralized in other blogposts and docs that I've read.

Reply:
Revert. I wasn't sure about that. Best to be consistent. The wording is a bit clumsy.

Reply:
I agree, reverted